src/apps/myaccount/submodules/billing/billing.js

```diff
@@ -161,12 +177,37 @@ define(function(require) {
 				if (!settingsItem.hasClass('open')) {
 					settingsItem.find('input').keyup();
 				}
+
+				template.find('.credit-card-container').show();
+			});
+
+			template.on('change', '#dropin_container', function(e) {
+				console.log('changed');
```

Comment:
remove this, also, is this whole block of code needed?

Reply:
good point, i forgot to remove it :(